sx-auth.el

```diff
@@ -47,32 +47,33 @@ what you are doing!
 
 This variable is set with `sx-auth-authenticate'.")
 
-(defvar sx-auth-method-auth '((me . t)
-                              (inbox . t)
-                              (notifications . t)
-                              (events . t)
-                              (posts (comments add))
-                              (comments delete
-                                        edit
-                                        flags
-                                        upvote)
-                              (answers accept
-                                       delete
-                                       downvote
-                                       edit
-                                       flags
-                                       upvote)
-                              (questions answers
-                                         add
-                                         close
-                                         delete
-                                         downvote
-                                         edit
-                                         favorite
-                                         flags
-                                         render
-                                         upvote
-                                         (unanswered my-tags)))
+(defconst sx-auth-method-auth
+  '((me . t)
+    (inbox . t)
+    (notifications . t)
+    (events . t)
+    (posts (comments add))
+    (comments delete
+              edit
+              flags
+              upvote)
+    (answers accept
+             delete
+             downvote
+             edit
+             flags
+             upvote)
+    (questions answers
+               add
+               close
+               delete
+               downvote
+               edit
+               favorite
+               flags
+               render
+               upvote
+               (unanswered my-tags)))
```

Comment:
Another note to self: see what can be done about using the tree splicer for this structure as well. Should be robust enough to handle it.